### ISSUE TITLE
checkra1n brings error when trying to install cydia

### ISSUE THREAD

**Issue by lox1318** [checkra1n/BugTracker#1854]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 7
2. On what version of iOS is it?
iOS 14.2
3. What version of checkra1n are you using?
0.12.0
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Macbook, macOS Catalina (10.51.1)
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
Official Apple Cable onto Macintosh‘s direct usb hub

**What are the steps to reproduce the issue?**
1.  Jailbreak
2. Get the checkra1n app on your home screen
3. Open app
4. press in cydia and install
5. check_snapshot_error_code: Unable to remount rootfs

**What do you expect, and what is happening instead?**

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Yes

**Any other info, error logs, screenshots, ...?**
check_snapshot_error_code: Unable to remount rootfs, read-write.

![25333250-E48F-44B7-B16E-A20364A2CDAD](https://user-images.githubusercontent.com/74625701/99446161-a6288b80-28ec-11eb-949d-d8df45e1f9bc.png)

**Comment by IamMoonUwU** [comment]
Hey I aslo have the same problem have you found a solution to it yet?

**Comment by lox1318** [comment]
Nope, i can’t jailbreak. i would wait until a semi untethered jailbreak for
ios 14 comes out
